documentation.yml

```
@@ -0,0 +1,56 @@
+#   html:
+#     path: "./docs"
+#     theme: "default"
+#   markdown:
+#     path: "./docs/markdown"
+#     toc: true
+
+# Documentation structure
+toc:
+  - name: Introduction
+    description: Overview of the application
+  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)